async function main() {
	//compile in the code
	//compile separatly with yarn
}

main()
	.then(() => process.exit(0))
	.catch((error) => {
		console.error(error);
		process.exit(1);
	});
